web/src/components/dashboard/SidebarCardsList.tsx

```diff
@@ -133,7 +141,7 @@ export const SidebarCardsList = (props: Props): ReactElement => {
           {showEmptyState() && (
             <div className="fdc fac margin-y-3">
               <h3 className="text-normal">{Config.dashboardDefaults.emptyOpportunitiesHeader}</h3>
```

Comment:
[boulder] This appears to be missing the AC for "And new text styling" such as changing this h3.

See the difference in text styling between the current version in dev:
![image](https://github.com/newjersey/navigator.business.nj.gov/assets/6563663/380d9c3a-ff7d-4bca-859e-b1c565259149)

and the new styling in the designs from the story:
![image](https://github.com/newjersey/navigator.business.nj.gov/assets/6563663/e3ef3c26-a038-4dcf-ae66-801ffe03cd56)

It doesn't just change the image; the text styling is also updating